crnk-setup/crnk-setup-rs/src/main/java/io/crnk/rs/CrnkFilter.java

```diff
@@ -42,6 +42,12 @@ public CrnkFilter(CrnkFeature feature) {
 
 	@Override
 	public void filter(ContainerRequestContext requestContext) {
+		if (feature.getWebPathPrefix() != null) {
+			String path = UrlUtils.removeLeadingSlash(requestContext.getUriInfo().getPath());
+			if (!path.startsWith(UrlUtils.addTrailingSlash(feature.getWebPathPrefix()))) {
```

Comment:
I guess it would make sense to add that trailing slash to `getWebPathPrefix`? Currently it does not. But there is no use case for that.

Reply:
it would ease the check here